main.nf

```diff
@@ -94,14 +94,91 @@ if (params.help) {
 }
 
 // Validate inputs
-if(!params.raw_input){
-   params.mzmls = params.mzmls ?: { log.error "No spectra data privided. Make sure you have used the '--mzmls' option."; exit 1 }()
+//MS Input
+if (params.sample_sheet)  {
+   sample_sheet = file(params.sample_sheet)
+
+   Channel.from( sample_sheet )
+                .splitCsv(header: true, sep:'\t')
+                .map { col -> tuple("${col.ID}", "${col.Sample}", "${col.Condition}", file("${col.ReplicateFileName}", checkifExists: true))}
```

Comment:
in the doc you use Samples but there it is col.Sample
I would make this consistent (and maybe prefer "Sample" over "Samples" as column header)

Reply:
ok will change this

Reply:
is fixed now